### ISSUE TITLE
Some images not in srcset

### ISSUE THREAD

**Issue by Mellivora** [ResponsiveImagesCG/wp-tevko-responsive-images#24]
Hello, thanks very much for your work on this plugin.

I have some custom image sizes defined as well as Wordpress defaults. Only the Wordpress default thumbnail has a hard crop and I am only working with featured images at the moment.

Some image sizes (with a soft crop) were not being put in the srcset which appeared to be because of the comparison of the aspect ratios in the php (which looks like it is only supposed to remove hard cropped images?).

For one image $ratio is 1.34 but for the Wordpress medium and large image sizes for that image $crop_ratio is 1.33 and they were being unset from $default_sizes. I altered the code to work out the aspect ratios to:

```php
$ratio = floor( ( $width / $height ) * 10 );
foreach ( $default_sizes as $key => $image_size ) {
    $crop_ratio = floor( ( $image_size['width'] / $image_size['height'] ) * 10 );
```
That seems to solve the problem for me and now the $ratio and $crop_ratio for my images match apart from the thumbnail which has a hard crop and gets unset. Is this how it is supposed to work?

**Comment by joemcgill** [comment]
Thanks @Mellivora. I want to make sure the way the ratio is rounded catches as many edge cases as possible. Could you tell me what the settings are for the custom size you've created that is being rounded to 1.34?

Thanks,
Joe

**Comment by joemcgill** [comment]
@tevko can you reopen this issue? This fix was revered in #25.

**Comment by Mellivora** [comment]
Hi @joemcgill, thanks for your reply.

My image size that is being rounded to 1.34 is a 75px wide thumbnail which was being created from a 1600 x 1200 original and that was the only size that actually got rounded wrong for that image.

A better example is a 562 x 250px image I am using and the medium and 75px thumbnail sizes get rounded differently.

```php
$ratio: 2.25, $width: 562, $height: 250

thumbnail - $crop_ratio: 1, $image_size['width']: 150, $image_size['height']:150

medium - $crop_ratio: 2.26, $image_size['width']: 300, $image_size['height']:133

seven-five-thumb - $crop_ratio: 2.27, $image_size['width']: 75, $image_size['height']:33

randt-large - $crop_ratio: 2.25, $image_size['width']: 562, $image_size['height']:250

full - $crop_ratio: 2.25, $image_size['width']: 562, $image_size['height']:250
```
